frontend/web/components/AdminAPIKeys.js

```diff
@@ -190,7 +190,7 @@ export class CreateAPIKey extends PureComponent {
                 <Input
                   ref={(v) => {
                     setTimeout(() => {
-                      v.focus()
+                      v?.focus()
```

Comment:
This change probably doesn't belong in this PR?

Reply:
This can be moved to another PR